stories/10-packages/Loadable.stories.mdx

```diff
@@ -0,0 +1,34 @@
+<Meta title="Packages|Loadable" />
+
+# Loadable
+This package adds functionality to Irving core for using the [`@loadable/component`](https://loadable-components.com/) package.
+
+## Installation
+1. `npm install @irvingjs/loadable`
+2. Add the `@irvingjs/loadable` config to your project's `irving.config.js`:
+
+```javascript
+// Import it from the package.
+import loadableConfig from '@irvingjs/loadble';
+
+// Add it to your irving config.
+export default {
+  packages: [
+      loadableConfig,
+  ],
+};
+```
+
+## Using this package
+Once configured, you can use any feature of the `@loadable/component` package. See the [API Docs](https://loadable-components.com/docs/api-loadable-component/) for more information. The most basic way to use it, however, is to modify your component map to use loadable components instead of statically-imported ones. Example:
+
+```javascript
+// componentMap.js
+import loadable from '@loadable/component';
+
+const componentMap = {
+    'my-project/test': loadable(() => import('./components/test')),
```

Comment:
Is there some way we could offer a version of the styled components package that does this automatically?

Immediate thoughts on possibilities,
* Another export from `styled-components` for `createThemedAndLoadableComponentMap`
* An irving.config value that flags it to be true `{ useLoadableStyledComponents: true, }`

Reply:
Yeah I could see just making a separate component map for this.